include/vara/Feature/FeatureModelTransaction.h

```diff
@@ -803,14 +861,19 @@ class FeatureModelCopyTransactionBase {
   }
 
   Constraint *addConstraintImpl(std::unique_ptr<Constraint> NewConstraint) {
-    if (!FM) {
-      return nullptr;
-    }
+    assert(FM && "FeatureModel is null.");
```

Comment:
Do we want to just assert here or return by default a nullptr when we are called with no FM?
@s9latimm What do you think?

Reply:
I had a look at other Transactions. It's handled in three ways: check and return nullptr, assert, no check. I think this should be solved after error codes are implemented.

Reply:
Yes, error codes should make this case clear, so let's keep it that way.